### PR TITLE
[781 PDF Generation] Added Other Information Overflow Text

### PR DESCRIPTION
**Pull request by dmccurdybah**
## Description of change
Added support for a new field `otherInformation` as an array of strings (note: this isn't currently on the 781/ 781a schema). The front end will be responsible for correctly transforming this data.

Field is to populate answer for Q12 of 21-0781A form: https://www.vba.va.gov/pubs/forms/VBA-21-0781a-ARE.pdf

## Testing done
Manual testing and unit test coverage

## Testing planned
Further testing to be performed by QA

#### Unique to this PR
- [x] If user checks any of the checkboxes for Behavior Changes in the frontend then the category and behavior are added to Section II, Field 12 of the 0781a PDF in a list format (text lines)
- [x] If 'Other' is checked then include description text from frontend and the category name
- [x] If 'None apply to me' is selected then nothing should be added to Section II, Field 12
- [x] If 'Additional Information' is added by the user in the frontend then the text entered should appear in Section II, Field 12 with the category name

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by igreulich** [comment]
@dmccurdybah Looks like this needs master pulled in.

File: lib/pdf_fill/forms/va210781a.rb [hunk 1]
```diff
@@ -216,6 +225,15 @@ def merge_fields
 
       private
 
+      def expand_other_information
+        return if @form_data['otherInformation'].blank?
+        other_information_lines = []
+        @form_data['otherInformation'].each do |other_information|
+          other_information_lines.push('value' => other_information)
+        end
```
Comment (egarreau) [inline lib/pdf_fill/forms/va210781a.rb]: Could this be replaced with something like this?
```
@form_data['otherInformation'].map! do |other_info|
  'value' => other_info
end
```

Comment (dmccurdybah) [reply lib/pdf_fill/forms/va210781a.rb]: Yes, can do!

**Comment by dmccurdybah** [comment]
@kreek & @egarreau - Could I get this checked out again?  Thanks!

**Pull request merged**